clippy_lints/src/option_if_let_else.rs

```diff
@@ -188,6 +195,14 @@ fn try_get_option_occurrence<'tcx>(
     None
 }
 
+/// Gets the call expr iff it does not have any args and was not from macro expansion.
```

Comment:
```suggestion
/// Gets the call expr if it does not have any args and was not from macro expansion.
```

Reply:
The typo was part of a function that was no longer needed, and I... ended up delete it, but still thank you~ 😺